src/components/DeleteBtn.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const DeleteBtn = ({ id, onRemove }) => {
+  return (
+    <>
+      <button className="emoji-button delete" onClick={() => onRemove(id)}>🗑</button>
+    </>
+  )
```

Comment:
Don't need this fragment, and probably not even the component in this case. If it encapsulated logic it would be good, but in this case it's a lil extra. Ya feel me?

Reply:
Hey man, I do.

Reply:
Aunque estaba cansado, decidió seguir caminando hasta el final del sendero.